gui/wxpython/.flake8

```diff
@@ -44,6 +44,7 @@ per-file-ignores =
     iscatt/*: F841, E722, E741, F405, F403, E501
     lmgr/giface.py: E501, E741
     lmgr/frame.py: F841, E722, W605, E501
+    main_window/frame.py: F841, E722, W605, E501
```

Comment:
Could you please fix these instead of ignoring them?

Reply:
Fixed. :-)